bftengine/include/bftengine/ReservedPagesClient.hpp

```diff
@@ -81,6 +81,16 @@ class ResPagesClient : public ReservedPagesClientBase, public IReservedPages {
     }
   }
 
+  static uint32_t getNumResPages() {
```

Comment:
a bit confusing name, since the it should return client's num of pages

Reply:
Any suggestion for a better one?

Reply:
there is already a not very successful `numberOfReservedPages()` name...
maybe getMyNumResPaged() :-/

Reply:
I was thinking that `ResPagesClient::numberOfReservedPages()`'s return value is misleading in the sense that it returns the total number of reserved pages, if I am not mistaken (because it will have a pointer to the one initialised in ST). Maybe it should return the "client/my" value instead?

Maybe we can name the static function something like `numberOfReservedPagesForClient()` or go with `getMyNumResPages()`, I don' have a preference.

Reply:
just choose

Reply:
Done.

Reply:
Since we are on the topic. What should `IRservervedPages:: numberOfReservedPages()` return, in general>?

Reply:
the total number

Reply:
In that case, I guess we can remove it from the interface and use the information from the registry (separately from this PR).